Kolskid standomix300 make bread

dmbrasw2 opened the pull request:
## Completed Work

- Created new procedure: 'p-standomix300-make-bread'
- Procedure's Goal: to make mersandwich bread using the Stand'o'mix 300
- Laid out the basic sections and added first draft of steps
- Added images to repository and procedure

## User Scenario

"Hannah started baking bread during the pandemic as a hobby so she could feel productive while isolating. She enjoys having control over the ingredients and flavors, her homemade bread is generally healthier than mass-produced store bread, and she enjoys having fresh, homemade bread every week. Kneading the dough is time consuming and requires a lot of physical effort, and when she injured her wrist she was unable to make bread for two months. Hannah wants to make the whole process easier, and avoid having to take such a long break from baking again, by using a stand mixer to mix and knead the dough for her."

## Notable Review Requests:

- There are a few steps that felt to short to be their own thing, so I combined them with the next short step. Technically it may be doing two things at once, but at the same time dividing "add ingredients to bowl" and "mix ingredients" into two separate steps feels awkward. 

- The section about pre-measuring and dividing out the flour: I tried my best to make it clear, but I'm worried doesn't read well. Any suggestions about how to rewrite/improve/move it would be appreciated! [You can jump right to it here](https://github.com/ENG517/Procedure/commit/2d5a23ee5262e6eef6d0f48be8f9d8e09b30e875#r165620516)

- Question about additional information [re: preheating the oven](https://github.com/ENG517/Procedure/commit/2d5a23ee5262e6eef6d0f48be8f9d8e09b30e875#r165620635)

Ademola-Adepoju commented:
This looks good, Sami!

I think it's pretty much the same comment I had on the first one.

Ademola-Adepoju reviewed (approved):
I already left a review but normally, I think this is good to go.

Ademola-Adepoju reviewed (approved):
Sweet. I like it. Additional comment to what I already mentioned before is I like how you also sub-headed the steps. It also validates my decision to do something similar in my own document.

dmbrasw2 committed:
Initial setup

Copied and renamed TOCOPY

dmbrasw2 committed:
Made branch 3

Renamed task 3 to make bread and made a make bread branch

dmbrasw2 committed:
Began writing brad procedure

Added intro, required tool, and ingredients

dmbrasw2 committed:
Finished draft of bread and made pull request

Finished the bread procedure and added pictures

Created and filled out the pull request

dmbrasw2 committed:
Fixed pic placement and bullet indents

Fixed placement of doughball pic.
Resized bloomed yeast pic (smaller) and added alt text. Found potential alternate blooming yeast pic, but not using yet.

Indented bullets under/with steps

dmbrasw2 committed:
Added another picture, tweaked 'rise again' phrasing

Added a pic of proofed dough in loaf pan

Revised and reworked the steps for shaping the dough into a log, and added a picture.

Changed the format & title of the section explaining why to preheat the oven for so long.

dmbrasw2 committed:
Fixed some writing; fixed picture alignment

The spacing between some steps is larger than others ('Make the dough' vs 'Rise Again') and I don't know why.

Deleted a half-deleted line that got left in a step

Fixed image alignment, including moving the doughball picture so it isn't tacked onto the end of the "the dough should be smooth" line

Added soft breaks to some sections

gmsimon22 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -20,10 +20,63 @@ Baking bread is a fun and therapeutic activity with a bonus: you get homemade, f
 - 5 ½ cups all-purpose or bread flour
 
```
Maybe add a difficulty level somewhere in the beginning. Could be a helpful addition or reference for bakers.
Ex: 
Difficulty Level: Beginner
Beginner Friendly!
etc.

gmsimon22 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -20,10 +20,63 @@ Baking bread is a fun and therapeutic activity with a bonus: you get homemade, f
 - 5 ½ cups all-purpose or bread flour
 
  ### *About Ingredients…*
```
Ooh I love this section. Very helpful to know what the ingredients are for and why

gmsimon22 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -20,10 +20,63 @@ Baking bread is a fun and therapeutic activity with a bonus: you get homemade, f
+- Testing for doneness: Gently (carefully!) tap the top of a loaf: it should sound hollow.
+2. Turn the loaves out onto a wire cooling rack and allow to cool at least 10 minutes before slicing.
+- Optional: while the loaves are still hot, brush the tops with butter.
+3. Once cool, store in an airtight container or bag for 2-3 days at room temperature, or up to 5 days in the refrigerator.
```
Maybe add some pictures somewhere toward the end since the last 3 subheadings don't have any visual aids

alyssajohnsonunofficial replied:
I think Gabby's idea could be good for adding even more visual appeal!

gmsimon22 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -20,10 +20,63 @@ Baking bread is a fun and therapeutic activity with a bonus: you get homemade, f
+5. Cover the pans with a dishtowel, and allow to rise for another 45 minutes - 1 hour, or until they’ve risen about 1 inch above the loaf pan.
+6. Preheat the oven to 350 F, either right away or at least 20 minutes before the bread is finished rising.
+
+    > **The importance of a long preheat**: _The sensors in your oven only track the temperature of the **air**, which is going to rush out when you open the door to put the bread in. This results in the oven taking longer to return to temperature at the start of your bake._
```
great added tips!

gmsimon22 reviewed (commented):
Amazing job! I added my comments, but I think you did a really great job on this one.

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+# How to Make Sandwich Bread
+
+Baking bread is a fun and therapeutic activity with a bonus: you get homemade, fresh bread with no additives, preservatives, or excess sugar. And it’s even easier to make when you don’t have to knead the dough by hand or cover your counters in flour.
+
```
I like the staging/motivation here!

alyssajohnsonunofficial replied:
Agreed, this is a really strong intro! It definitely sets up the intended audience and why they need this guide!

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+- Large bowl
+- Oven
+- Two loaf pans
+
```
Because the Stand'o'Mix 300 is a pretty big prerequisite, I'd include something about it in the motivation. Such as why it's better for kneading bread than using your hands.

alyssajohnsonunofficial replied:
Agreed, definitely think it would be good to include something justifying the mixer's use!

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+- 2 tsp salt
+- 2 tbsp neutral oil (canola or vegetable) 
+- 5 ½ cups all-purpose or bread flour
+
```
How many loafs are we making?

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+> - **Flour**: For this recipe, bread and all-purpose flour are interchangeable. The only difference is that the extra protein in bread flour will produce a slightly chewier loaf.  
+> - **Yeast**: Active Dry Yeast is not the same as Instant or Rapid Rise yeast, and they all behave differently. The instructions and time estimates in this recipe are intended for active dry yeast.  
+> - **Oil**: Oils with strong flavors, like olive oil, will affect the taste of the final loaf. Neutral oils will be undetectable and give the best, classic bread flavor.
+
```
Great information here! I'd consider adding something like this to the other procedures as well.

alyssajohnsonunofficial replied:
Definitely a great addition! Could be interesting to see it in the other procedures for sure.

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+5. Measure and divide the flour into: 3 cups, 1 cup, and the remaining 1½ cup. 
+   - The flour will be added in 3 stages, not all at once: first 3 cups, then 1 cup, and then however much more is needed to form a smooth, slightly-tacky dough that pulls away from the sides of the bowl;
+   - The exact amount of flour the dough needs will vary depending on the humidity in your kitchen.
+
```
Love your use of unordered lists here! I think it breaks up the tasks quite nicely.

alyssajohnsonunofficial replied:
Agreed! Looks very visually appealing and makes it easier for the reader to take in all the info

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+4. Dust a clean area of counter with flour, and lay out one of the halves.
+5. Press and stretch the dough into a long, flat rectangle roughly as wide as the loaf pan is long.
+   - Don't worry about being too rough or popping air bubbles. Some people use a rolling pin for this step!
+
```
Rolling pin = alternative path!

WBlan97 commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,94 @@
+   - Testing for doneness: Gently (carefully!) tap the top of a loaf: it should sound hollow.
+2. Turn the loaves out onto a wire cooling rack and allow to cool at least 10 minutes before slicing.
+   - Optional: while the loaves are still hot, brush the tops with butter.
+3. Once cool, store in an airtight container or bag for 2-3 days at room temperature, or up to 5 days in the refrigerator.
```
Is there a concrete way to know when it's ready?

WBlan97 commented:
Really liked your use of images on this one. I don't think Hannah should have any problem making this bread.

dmbrasw2 committed:
Made edits per reviews

Made some general edits per peer reviews

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,95 @@
+
+### Ingredients for 2 Loaves
+- 2 cups warm water (105-115F)
+- 1 Tablespoons active dry yeast
```
Just a note that I'd make this '1 tablespoon'; corrects the grammar issue and also keeping it lowercase makes it consistent with the rest of the list!

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,95 @@
+
+## Process
+
+_Don’t forget: Baking is as much art as science! Things like humidity and the temperature of the room will affect the dough in different ways, and no two doughs will be exactly the same. Some might need more water, some might need more flour. Go with the flow, and don't sweat it!_
```
This is a great note to include! Really helps tailor this to your intended audience of beginner bakers!

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,95 @@
+   - Allow each addition to fully incorporate before adding more.
+   - The dough should be smooth, elastic, and lightly tacky to the touch. If the dough is sticky, add a little more flour. 
+            
+    ![The doughball is cohesive, smooth, and pulls away from the mixing bowl.](<assets/images/screenshots/Bread pics/Bread dough.webp>)
```
Great use of images here! This really adds to the helpfulness

alyssajohnsonunofficial commented on kolskid/p-standomix300-make-bread.md:
```diff
@@ -0,0 +1,95 @@
+![alt text](<assets/images/screenshots/Bread pics/Proofed in loaf pan.png>)
+
+
+**Why such a long preheat?**   
```
Another great addition, also super helpful

alyssajohnsonunofficial reviewed (commented):
This one looks great too! Really great job making these informative yet interesting, and tailoring them appropriately to the intended audience!

dmbrasw2 committed:
Spelling Corrections + Added a pic

Corrected spelling and added a picture at the end per peer review

dmbrasw2 committed:
Update p-standomix300-make-bread.md

dmbrasw2 committed:
Staging edits

Reworded staging section

dmbrasw2 committed:
Phrasing and spacing

Edited phrasing of checking doneness with a thermometer

Fixed uneven/different list spacing

dmbrasw2 committed:
User scenario: Make bread

Copied user scenario.md file, renamed 'User Scenario Make Bread'. Will do the same for other branches

dmbrasw2 committed:
Added 'figure'

Added figure and description to all images, plus alt text to a few images that were missing it.

dmbrasw2 committed:
Fixed alignment

Fixed 'Figure 2' alignment with image

dmbrasw2 committed:
User Scenario + Task Analysis doc

Added user scenario for make bread

Added PDF of Task Analysis to the docs folder

dmbrasw2 committed:
User scenario fix

Copied user scenario wrong the first time. Deleted the miscopied file.

dmbrasw2 committed:
Added user scenario

Added and filled out user scenario for make bread

dmbrasw2 committed:
Last minute tweaks + Reflection

Made a tweak to the formatting of 'Why such a long preheat?'

Wrote the Reflection in the Assets folder.

dmbrasw2 committed:
Reflection edits

dmbrasw2 committed:
Reflection update

Wrote more

lingeringcode committed:
Feedback for Deanna

dmbrasw2 committed:
Applied changes per feedback

Numbered headings, reworded some headings, fixed some grammar errors
